examples/v2/project_creation/apis.py

```diff
@@ -19,14 +19,19 @@ def GenerateConfig(context):
 
   project_id = context.properties['project']
   billing = context.properties['billing']
+  concurrent_api_activation = context.properties['concurrent_api_activation']
 
   resources = []
-  for api in context.properties['apis']:
+  for index, api in enumerate(context.properties['apis']):
+    dependsOn = [project_id, billing]
+    # Serialize the activation of all the apis by making api_n depend on api_n-1
```

Comment:
Do you think it would be useful to still do a few api's concurrently? Like 4 at a time, or so? Could be significantly faster, and might not exceed quota?